Cargo.toml

```diff
@@ -1,24 +1,26 @@
 [package]
 name = "parity-scale-codec"
 description = "SCALE - Simple Concatenating Aggregated Little Endians"
-version = "3.6.8"
+version = "3.7.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "Apache-2.0"
 repository = "https://github.com/paritytech/parity-scale-codec"
 categories = ["encoding"]
 edition = "2021"
+build = "build.rs"
 rust-version = "1.60.0"
 
 [dependencies]
 arrayvec = { version = "0.7", default-features = false }
 serde = { version = "1.0.204", default-features = false, optional = true }
 parity-scale-codec-derive = { path = "derive", version = ">= 3.6.8", default-features = false, optional = true }
-bitvec = { version = "1", default-features = false, features = [ "alloc" ], optional = true }
+bitvec = { version = "1", default-features = false, features = ["alloc"], optional = true }
 bytes = { version = "1", default-features = false, optional = true }
 byte-slice-cast = { version = "1.2.2", default-features = false }
 generic-array = { version = "0.14.7", optional = true }
 arbitrary = { version = "1.3.2", features = ["derive"], optional = true }
 impl-trait-for-tuples = "0.2.2"
+rustversion = "1.0.17"
```

Comment:
We don't need it for the normal dependencies?

Reply:
You're right. Missed it. Done.